tst/standard/main.tst

```diff
@@ -0,0 +1,29 @@
+gap> A := NullMat(3, 3, GF(19));;
+gap> EchelonMatBlockwise(A);
+rec( heads := [ 0, 0, 0 ], vectors := [  ] )
```

Comment:
Let's also check here that `coeffs` and `relations` are not bound.

Reply:
ATM this test may not make the most sense, but once we make `EchelonMat{Transformation}Blockwise` into more then dummy functions this will make more sense.